functions/slack/requirements.txt

```
@@ -1,4 +1,4 @@
 google-api-python-client==2.131.0
 flask==3.0.3
-functions-framework==3.5.0
+functions-framework==3.8.2
```

Slack sample is missing `KG_API_KEY`, remove for now

```suggestion
functions-framework==3.5.0
```